cli/src/test/java/com/devonfw/tools/ide/tool/tomcat/TomcatTest.java

```diff
@@ -55,7 +55,6 @@ public void testTomcat() {
                 + " Therefore, we install a compatible version in that range."),
         new IdeLogEntry(IdeLogLevel.DEBUG, "Installed java in version " + javaVersionTomcat + " at ", true),
         new IdeLogEntry(IdeLogLevel.SUCCESS, "Successfully installed tomcat in version " + tomcatVersion),
-        new IdeLogEntry(IdeLogLevel.INFO, "OpenJDK version " + javaVersionTomcat),
```

Comment:
why was this removed? Did this fail after your changes?
Does it reveal a bug or are we sure this is OK?